index.js

```diff
@@ -40,7 +40,48 @@ function toggleColor(element) {
   }
 }
 
+//find dom object
+const header = document.querySelector('#header')
+//add listener to object
+header.addEventListener('click', function(){
+  toggleColor(header)
+  // click
+  // work
+})
 
 /***** Deliverable 2 *****/
+//find form object
+const form = document.querySelector('#new-player-form')
+const submitBtn = form.querySelector('[type=submit]')
+
+form.addEventListener('submit', function(e){
+  let player = {}
+  const formInfo = e.target
+  player["number"] = formInfo.number.value
+  player["name"] = formInfo.name.value 
+  player["nickname"] = formInfo.nickname.value
+  player["photo"] = formInfo.photo.value
+  player["likes"] = 0
```

Comment:
Looks good! Try to use the dot notation for objects, it's a more common syntax among Javascript developers. Also, if you want to shorten this code up a bit, you can create the `player` object and assign values to it all at once:

```js
const player = {
    number: formInfo.number.value,
    name: formInfo.name.value,
    nickname: formInfo.nickname.value,
    photo: formInfo.photo.value,
    likes: 0
}
```